guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -6,9 +6,8 @@ Use this procedure to enable the repositories that are required to install {Prod
 
 ifdef::foreman-el,katello[]
 * xref:#repositories-centos-8[CentOS 8]
-* xref:#repositories-centos-7[CentOS 7]
 endif::[]
-ifdef::foreman-el,katello,satellite[]
+ifdef::satellite[]
```

Comment:
I'm not sure this is needed. It's not like Satellite is going to support EL7 either.

Reply:
Removed!